workflow/scripts/parallel_vcf.py

```diff
@@ -0,0 +1,50 @@
+import os
+import argparse
+from multiprocessing import Pool
+from pysam import VariantFile
+import tempfile
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument("file_vcf")
+parser.add_argument("command")
+parser.add_argument("--stepsize", "-s", type=int, default=1000000)
+parser.add_argument("--threads", "-t", type=int, default=8)
+parser.add_argument("--tmp", default=None, help="Define the base directory for the tmp files")
+args = parser.parse_args()
+tmp_base = args.tmp
+stepsize = args.stepsize
+
+# create regions
+f = VariantFile(args.file_vcf)
+regions = []
+for name, contig in f.header.contigs.items():
+    last = 0
+    for x in list(range(0, contig.length, stepsize)):
+        stop = min(last + stepsize - 1, contig.length)
+        regions.append(f"{name}:{last}-{stop}")
+        last = stop+1
+f.close() 
+
+#command = "SnpSift annotate {ss_args} {args.database_vcf} /dev/stdin"
+command = args.command
+
+with tempfile.TemporaryDirectory(dir=tmp_base) as t:
+    def f(x):
```

Comment:
Please use readable variable and function names.

Reply:
jub